File: Testes de performance por disciplina/Fundamentos de Desenvolvimento com C#/3º Teste de Performance/questao_01/Book.cs
namespace TesteDePerformance3.questao_1;

public class Book{
    private string _name;
    private string _author;
    private string _genre;
    private int _releaseYear;

    public Book(string name, string author, string genre, int releaseYear){
        _name = name;
        _author = author;
        _genre = genre;
        _releaseYear = releaseYear;
    }

    public string GetBookInfo(){
        return $"Nome: {_name}\n\tAutor: {_author}\n\tGênero: {_genre}\n\tRelease Year: {_releaseYear}\n";
    }
}

File: Testes de performance por disciplina/Fundamentos de Desenvolvimento com Java (P2)/3º Teste de Performance/questao_01/Book.java
package com.bam.sandcastlejava.edu.infnet.tp3.questao_01;

public class Book {
    private String title;
    private String author;
    private String genre;
    private int releaseYear;

    public Book() {}

    public Book(String title, String author, String genre, int releaseYear) {
        this.title = title;
        this.author = author;
        this.genre = genre;
        this.releaseYear = releaseYear;
    }

    @Override
    public String toString() {
        return "Livro:\n\t%s\n\tAutor: %s\n\tGênero: %s\n\tAno de lançamento: %d\n".formatted(title, author, genre, releaseYear);
    }
}